widgetsbase/proguard-rules.pro~HEAD

```diff
@@ -0,0 +1,21 @@
+# Add project specific ProGuard rules here.
```

Comment:
There are 4 .gitignore and 4 proguard-rules.pro files with strange names, like: HEAD, task-57-reorganize-widgets-classes, ...
Remove them, pls.

Reply:
Done